Library/Formula/pgcli.rb

```diff
@@ -0,0 +1,65 @@
+class Pgcli < Formula
+  homepage "http://pgcli.com/"
+  url "https://pypi.python.org/packages/source/p/pgcli/pgcli-0.10.2.tar.gz"
+  sha1 "c34330335e08418aba1c406ac846b37daaf21262"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "openssl"
+  depends_on :postgresql
+
+  resource "click" do
+    url "https://pypi.python.org/packages/source/c/click/click-3.3.tar.gz"
+    sha1 "d716a932b930d71059e49465b6b42e833808369a"
+  end
```

Comment:
Thanks for changing this; can you put an additional newline between each `resource` block?